### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by montaniasystemab** [soliantconsulting/SimpleFM#101]
Hello

It seems that this bundle can no longer be installed from scratch since the package `litipk/php-exceptions` has been removed from github

```
  - Installing litipk/php-exceptions (0.1.3): Downloading (connecting...)
   Downloading (failed)       
   Failed to download litipk/php-exceptions from dist: The "https://api.github.com/repos/Litipk/php-exceptions/zipball/6271c3d5c51dab8bb64e4ac796b018574b477750" file could not be downloaded (HTTP/1.1 404 Not Found)
    Now trying to download from source
  - Installing litipk/php-exceptions (0.1.3): Cloning 6271c3d5c5

                                                                                                             
  [RuntimeException]                                                                                         
  Failed to clone https://github.com/Litipk/php-exceptions.git via https, ssh protocols, aborting.   
```

To me it seems like switching to version 0.8 of php-bignumbers would solve the issue but I'm not sure if it's entirely compatible

**Comment by jeremiahsmall** [comment]
Yes, this just bit us too. Working on a solution.

**Comment by jeremiahsmall** [comment]
We've fixed the dependency with the 5.1.3 tag.
